UI changes coming in v7.0 (#5693)
mattermost/docs: source/channels/share-links.rst

  :target: https://mattermost.com/deploy
  :alt: Available for Mattermost Self-Hosted deployments.

.. |copy-link-icon| image:: ../images/paperclip_F03E2.svg
  :height: 24px
  :width: 24px
  :alt: Copy the public URL link for an image in a message using the Copy link icon.

> You can share links to messages in Mattermost by selecting the **More Actions** option next to a message, then selecting **Copy Link**, or by hovering over an image and selecting the |copy-link-icon| icon in the top right corner. Paste the link into a message to share the image link with others. Sharing links to messages generates a preview of the message which helps minimize context switching. 

Reviewer: ```suggestion
To share links to messages in Mattermost, select the **More Actions** option next to a message, then select **Copy Link**. You can also hover over an image and select the |copy-link-icon| icon in the top right corner. Paste the link into a message to share the image link with others. Sharing links to messages generates a preview of the message which helps minimize context switching.
```
